Bioinfo_joint/bio_files_processor.py

```diff
@@ -0,0 +1,59 @@
+import os
+
+
+def convert_multiline_fasta_to_oneline(
+    path_to_directory: str,
+    input_filename: str,
+    output_filename: str,
+) -> None:
+    """Convert multiline FASTA to oneline and save as new file."""
+
+    os.chdir(path_to_directory)
+    fastq_file = os.path.join(input_filename)
+    fastq_file_out = os.path.join(output_filename)
+
+    elongation_line = ""
+    current_line = ""
+    write_to_file = ""
+    with open(fastq_file, "r") as outfile:

```

Comment:
Думаю, что в данном случае, более информативно будет назвать переменную не outfile а input_file. Поскольку в 12 строке `fastq_file`  определяется с использованием `input_filename` аргумента. Аналогично с названием переменных ниже
```suggestion
    with open(fastq_file, "r") as input_file:
```